datamodel/initdb.d/05_reference_implementation_support.sql

```diff
@@ -184,6 +181,26 @@ CREATE TABLE dcsa_im_v3_0.event_subscription_event_types (
     PRIMARY KEY (subscription_id, event_type)
 );
 
+DROP TABLE IF EXISTS dcsa_im_v3_0.event_subscription_shipment_event_type CASCADE;
+CREATE TABLE dcsa_im_v3_0.event_subscription_shipment_event_type (
+    subscription_id uuid NOT NULL REFERENCES dcsa_im_v3_0.event_subscription (subscription_id) ON DELETE CASCADE,
+    shipment_event_type_code varchar(4) REFERENCES dcsa_im_v3_0.shipment_event_type (shipment_event_type_code),
+    PRIMARY KEY (subscription_id, shipment_event_type_code)
+);
+
+DROP TABLE IF EXISTS dcsa_im_v3_0.event_subscription_transport_event_type CASCADE;
+CREATE TABLE dcsa_im_v3_0.event_subscription_transport_event_type (
+    subscription_id uuid NOT NULL REFERENCES dcsa_im_v3_0.event_subscription (subscription_id) ON DELETE CASCADE,
+    transport_event_type_code varchar(4) REFERENCES dcsa_im_v3_0.transport_event_type (transport_event_type_code),
+    PRIMARY KEY (subscription_id, transport_event_type_code)
+);
+
+DROP TABLE IF EXISTS dcsa_im_v3_0.event_subscription_equipment_event_type CASCADE;
+CREATE TABLE dcsa_im_v3_0.event_subscription_equipment_event_type (
+    subscription_id uuid NOT NULL REFERENCES dcsa_im_v3_0.event_subscription (subscription_id) ON DELETE CASCADE,
+    equipment_event_type_code varchar(4) REFERENCES dcsa_im_v3_0.equipment_event_type (equipment_event_type_code),
+    PRIMARY KEY (subscription_id, equipment_event_type_code)
+);

```

Comment:
Don't we need something similar for operations_event_type_code?

Reply:
yup just added it.